google/cloud/spanner/integration_tests/data_types_integration_test.cc

```diff
@@ -431,8 +431,8 @@ TEST_F(DataTypeIntegrationTest, InsertAndQueryWithStruct) {
 
 // Verify maximum JSON nesting.
 TEST_F(DataTypeIntegrationTest, JsonMaxNesting) {
-  // The default value of the backend max-nesting-level flag.
-  int const k_spanner_json_max_nesting_level = 90;
+  // The default value of the backend spanner_json_max_nesting_level flag.
+  int const k_spanner_json_max_nesting_level = 80;
```

Comment:
The name looks funny, if this is intended to be a compile-time constant maybe:

```
auto constexpr kSpannerJsonMaxNestingLevel = 80;
```

Reply:
Yeah ... without the `constexpr` that is what's required (I believe).  But I was happy with that as "spanner_json_max_nesting_level" is the true name of the Spanner flag, so it helped when I needed to re-search.

That said, see below.

Reply:
You can at least remove the `k_` prefix in that case.

Reply:
Done.